src/test/e2e/bundle_test.go

```diff
@@ -21,17 +21,11 @@ import (
 	"github.com/defenseunicorns/uds-cli/src/config"
 )
 
-func TestUDSCmd(t *testing.T) {
```

Comment:
was this test moved somewhere?

Reply:
no i just removed it cause i thought it wasn't useful. i see now though it was calling `uds` which should return the help so probably a good check to have after all.